src/renderers/WebGLRenderer.js

```diff
@@ -1056,9 +1072,11 @@ function WebGLRenderer( parameters ) {
 		// render scene
 
 		const opaqueObjects = currentRenderList.opaque;
+		const transmissiveObjects = currentRenderList.transmissive;
 		const transparentObjects = currentRenderList.transparent;
 
 		if ( opaqueObjects.length > 0 ) renderObjects( opaqueObjects, scene, camera );
+		if ( transmissiveObjects.length > 0 ) renderTransmissiveObjects( opaqueObjects, transmissiveObjects, scene, camera );
```

Comment:
Isn't there some way we could not have to render the opaque objects twice? It seems like we could just copy the previous buffer into the tranmissionSamplerMap.

Reply:
With WebGL2 we'll be able to avoid rendering twice, but with WebGL 1 we can't have antialias when rendering into a render target 😕

Reply:
Well, wouldn't we still be able to take the normal anti-aliased framebuffer we just rendered the opaque objects into and copy that into the `transmissionSamplerMap` for the next pass? Instead of doing it a second time? This would also remove the question around what size the samplerMap should be.

Reply:
Can WebGL 1 generate mipmaps for non-power-of two textures? The current implementation uses the lower level mips for roughness transmission.

Reply:
AFAIK, it can't.

Reply:
Ah yes, forgot about that. Could we sample it into a power-of-two texture, instead of a pure copy?